tools/jenkins/Jenkinsfile-PR

```diff
@@ -0,0 +1,19 @@
+pipeline {
+	environment {
+		ANDROID_HOME = '/opt/imint/android-sdk'
+		ANDROID_NDK_HOME = '/home/jenkins/android-ndk-r15c'
+		DISPLAY = ':0'
+	}
+	agent none // Force stages to specify agent
```

Comment:
Since there is only one label used, it is a tiny bit more elegant to replace `agent none` with the one below: `agent { label 'ooc' }` (and remove the one below).